src/views/Drawer/DrawerNavigatorItems.js

```diff
@@ -25,6 +25,7 @@ type Props = {
   inactiveTintColor?: string;
   inactiveBackgroundColor?: string;
   getLabelText: (scene: DrawerScene) => string;
+  getScreenConfig: (routeKey: string, configName: string) => string;
```

Comment:
I am sorry,
you mean passing this.props.router? if so how can i call getScreenConfig, i don't have childNavigationProps at DrawerNavigationItems.

When i call router.getScreenConfig(navigation, 'label') i always get the first route configuration.